### PR TITLE
feat: Add Thai Parliament's Public hearing (according to Thai constitution section 77)

### PR DESCRIPTION
**Pull request by itpcc**
## Involved Issue / 该 PR 相关 Issue

Close #

## Example for the Proposed Route(s) / 路由地址示例

```routes
/parliament/section77
```

## New RSS Route Checklist / 新 RSS 路由检查表
  
- [x] New Route / 新的路由
  - [x] Follows [v2 Script Standard](https://docs.rsshub.app/joinus/advanced/script-standard) / 跟随 [v2 路由规范](https://docs.rsshub.app/zh/joinus/advanced/script-standard)
- [x] Documentation / 文档说明
- [x] Full text / 全文获取
  - [x] Use cache / 使用缓存
- [ ] Anti-bot or rate limit / 反爬/频率限制
  - [ ] If yes, do your code reflect this sign? / 如果有, 是否有对应的措施?
- [x] [Date and time](https://docs.rsshub.app/joinus/advanced/pub-date) / [日期和时间](https://docs.rsshub.app/zh/joinus/advanced/pub-date)
  - [x] Parsed / 可以解析
  - [x] Correct time zone / 时区正确
- [ ] New package added / 添加了新的包
- [ ] `Puppeteer`

## Note / 说明

This PR is mainly for adding feed for Thai Parliament's Public hearing (according to Thai constitution section 77) system.

image

Due to issues with the parliament's firewall prevent `got` to request the page (either `socket hangs up`, or infinite redirect when using proxy), the module is forced to use Puppeteer to request the page instead. Note that the actual DOM traversing is still use cherrio as usual.

### CODE REVIEW

File: lib/v2/parliament/section77.js [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+                        title: item.find('a').text(),
+                        link: `${baseUrl}/${item.find('a').attr('href')}`,
+                        category: item
+                            .find('label')
+                            .toArray()
+                            .map((l) => $(l).text()),
+                    };
+                });
+        })
+    ).then((actPageList) => actPageList.reduce((actPage, actPageAggr) => actPageAggr.concat(actPage), []));
```
Comment (TonyRL) [inline lib/v2/parliament/section77.js:106]: Please extract data from the first page only.

File: lib/v2/parliament/section77.js [hunk 2]
```diff
@@ -0,0 +1,191 @@
...
+        });
+        await page.setContent((await response.buffer()).toString('utf8'));
+
+        // retrieve the HTML content of the page
+        const html = await page.content();
+        // close the tab
+        page.close();
+
+        return html;
+    };
```
Comment (TonyRL) [inline lib/v2/parliament/section77.js:63]: `got` still works fine if you do deal with the anti-crawling mechanism.
For requests without any valid cookies, the site will response with a 302 with cookie `__nxquid` and you have to use that in order to perform any further requests.
Either extract it from the set-cookie header or use CookieJar.

File: website/docs/routes/government.mdx [hunk 3]
```diff
@@ -84,6 +84,23 @@
 
 <Route author="itpcc" example="/dol/announce" path="/dol/announce/:owner?/:province?/:office?" paramsDesc={['Requester/former land owner', 'Province which the land is belongs to', 'DOL office name which the land is belongs to (สำนักงานที่ดิน(กรุงเทพมหานคร|จังหวัด*) [สาขา*])']} />
 
+## Thailand Parliament {#thailand-parliament}
+
+### Thailand Parliament Draft of Law's public hearing system {#thailand-parliament-thailand-parliament-draft-of-law-s-public-hearing-system}
+
+<Route author="itpcc" example="/parliament/section77" path="/parliament/section77/:type?" paramsDesc={['Type of hearing status, see below']} radar="1">
+  \| Presented by MP \*        | Presented by People \* | Hearing Ongoing       | Hearing ended    | Hearing result reported  | Waiting for PM approval | Assigned into the session | Processed  | PM Rejected |
+
```
Comment (TonyRL) [inline website/docs/routes/government.mdx:93]: Missing table header separators (`| --- | --- | ...`)

File: lib/v2/parliament/section77.js (hunk 1 above)
Comment (itpcc) [reply lib/v2/parliament/section77.js:106]: Sorry for late reply.

Is it okay to leave the page without description?

Comment (itpcc) [reply lib/v2/parliament/section77.js]: Oh, I see. Will remove additional pages then.

File: website/docs/routes/government.mdx [hunk 4]
```diff
@@ -84,6 +84,22 @@
...
 
+## Thailand Parliament {#thailand-parliament}
+
+### Thailand Parliament Draft of Law's public hearing system {#thailand-parliament-thailand-parliament-draft-of-law-s-public-hearing-system}
+
+<Route author="itpcc" example="/parliament/section77" path="/parliament/section77/:type?" paramsDesc={['Type of hearing status, see below']} radar="1">
+  \| Presented by MP \*        | Presented by People \* | Hearing Ongoing       | Hearing ended    | Hearing result reported  | Waiting for PM approval | Assigned into the session | Processed  | PM Rejected |
+  | เสนอโดยสมาชิกสภาผู้แทนราษฏร | เสนอโดยประชาชน  | กำลังเปิดรับฟังความคิดเห็น | ปิดรับฟังความคิดเห็น | รายงานผลการรับฟังความคิดเห็น | รอคำรับรองจากนายกรัฐมนตรี | บรรจุเข้าระเบียบวาระ | พิจารณาแล้ว  | นายกฯ ไม่รับรอง |
+  | ------------------------ | --------------- | ------------------- | --------------- | ------------------------ | --------------------- | ----------------- | ---------- | ------------- |
```
Comment (TonyRL) [inline website/docs/routes/government.mdx:94]: Markdown table can only contain one row of header.

File: lib/v2/parliament/radar.js [hunk 5]
```diff
@@ -0,0 +1,22 @@
+module.exports = {
+    'parliament.go.th': {
+        _name: 'รัฐสภา',
+        '.': [
+            {
+                title: 'รับฟังความคิดเห็นต่อร่าง พ.ร.บ. ตามมาตรา 77 ของรัฐธรรมนูญ',
+                docs: 'https://docs.rsshub.app/routes/government#thailand-parliament-section77',
```
Comment (TonyRL) [inline lib/v2/parliament/radar.js:7]:
```suggestion
                docs: 'https://docs.rsshub.app/routes/government#thailand-parliament',
```

**Pull request merged**